esterel-rhombus-lib/info.rkt

```diff
@@ -2,7 +2,7 @@
 
 (define collection 'multi)
 
-(define deps '("base" "rhombus-prototype" "esterel-lib"))
+(define deps '("base" "rhombus" "esterel-lib"))
```

Comment:
I think it is probably better to avoid depending on the meta-pkg and instead depend on the specific packages that are directly needed. IIUC, this dependency is actually equivalent to writing just `"rhombus-lib"` here, but it pulls in all of the other rhombus pkgs too.